skype/skype-ps/skype/Get-CsOnlineTelephoneNumber.md

```diff
@@ -155,7 +148,7 @@ Accept wildcard characters: False
 ```
 
 ### -ExpandLocation
-PARAMVALUE: SwitchParameter
+This parameter is reserved for internal Microsoft use.
```

Comment:
The ExpandLocation parameter here has the same behavior as in the next cmdlet, so you can use the same description.  It is not internal.

Reply:
Done.